fluentgen/src/main/java/com/azure/autorest/fluent/template/FluentManagerTemplate.java

```diff
@@ -173,6 +173,18 @@ public void write(FluentManager manager, FluentProject project, JavaFile javaFil
                 methodBlock.methodReturn("configure().authenticate(credential, profile)");
             });
 
+            classBlock.javadocComment(comment -> {
```

Comment:
This is for both premium and lite right?

Reply:
Yes, premium in PR https://github.com/Azure/azure-sdk-for-java/pull/27873